### PR TITLE
Add user data to sharing emails

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: server/mails/fr/mail_invitation.jade [hunk 1]
```diff
@@ -36,7 +36,7 @@ html(style="height:100%;")
             tr(style="padding-top: 10px; text-align: center; font-size: 0.85em; font-style: italic; color: #999;")
               td
                 p
-                  | Envoyé depuis le&nbsp;
+                  | Envoyé depuis le
```
Comment (frankrousseau) [inline server/mails/fr/mail_invitation.jade]: Sorry but you shouldn't remove the &nbsp; Most of our editors are configured to remove space located at end of lines.

**Comment by frankrousseau** [comment]
Thank you for this enhancement. Could you add comments and make smaller code blocks (I mean add blank lines more frequently)?

File: server/mails/fr/mail_invitation.jade (hunk 1 above)
Comment (m4dz) [inline server/mails/fr/mail_invitation.jade]: I think as long as the repo have an [editorconfig file](https://github.com/cozy/cozy-calendar/blob/master/.editorconfig#L15) that explicitely says that jade files must not be trimmed, It's OK to not use useless `&nbsp` that are unbreakables but rather use true whitespaces at end of line. The tools should not drive our code (and there is many editorconfig plugins for many IDEs :))

**Comment by m4dz** [comment]
I've added some improvements in code readability. Is it ok for you?

**Comment by frankrousseau** [comment]
Yep thank you!

File: server/mails/fr/mail_invitation.jade (hunk 1 above)
Comment (frankrousseau) [inline server/mails/fr/mail_invitation.jade]: Can you add this file to the current repository?

Comment (m4dz) [inline server/mails/fr/mail_invitation.jade]: It's already here :) https://github.com/cozy/cozy-calendar/blob/master/.editorconfig#L15

**Pull request merged**